homeassistant/components/proxmoxve/__init__.py

```diff
@@ -94,6 +96,9 @@ def setup(hass, config):
                 "Invalid credentials for proxmox instance %s:%d", host, port
             )
             continue
+        except SSLError:
+            _LOGGER.error("Unable to verify proxmox server SSL. try using \"verify_ssl: false\"")
```

Comment:
Start with capital letter after the period.

Reply:
```suggestion
            _LOGGER.error("Unable to verify proxmox server SSL. Try using \"verify_ssl: false\"")
```